### DISCUSSION TITLE
ESP32 Cam is very slow

### DISCUSSION THREAD

**Discussion by heindlalexander** [jomjol/AI-on-the-edge-device#2655] [Q&A] [answered]
I try to setup my device from scratch, but it's webinterface seems very slow, which makes it impssible to set it up.
the WiFi Signal is good (-60 dB), so I think it's not related to it.

**Comment by caco3** [comment]
please search the discussions for similar issues!

**Reply by heindlalexander** [reply, marked as answer]
good morning!
sorry, I searched for this problem, but I'm sure it is not related to WiFi Signal.
Meanwhile I recognized that it might be a problem with the SD card.
Files on it are damaged quite soon, so I guess it's worn out, and this may also affect the problem here...
I will try to use a new one.
